{{ cookiecutter.project_shortname }}/.github/workflows/tests.yml

```diff
@@ -1,95 +1,50 @@
+{% include "misc/header.py" %}
+
 {% raw -%}
 name: CI
 
 on:
   push:
     branches: master
   pull_request:
-    branches: master
+    branches:
+      - master
+      - "maint-**"
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '0 3 * * 6'
+    - cron:  "0 3 * * 6"
   workflow_dispatch:
     inputs:
       reason:
-        description: 'Reason'
+        description: "Reason"
         required: false
-        default: 'Manual trigger'
+        default: "Manual trigger"
 
 jobs:
   Tests:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          # You can add/remove combinations e.g. `dev` requirements or `postgresql13` by adding
-          # a new item to the following lists.
+          # TODO: You can add/remove combinations e.g. `dev` requirements or
+          # `postgresql13` by adding a new item to the following lists.
           # You can see the complete list of services and versions that are available at:
           # https://docker-services-cli.readthedocs.io/en/latest/configuration.html
-          python-version: [3.7, 3.8, 3.9, "3.10"]
-          requirements-level: [min, pypi]
-          cache-service: [redis]
-          db-service: [postgresql10, postgresql13, mysql5, mysql8]
-          mq-service: [rabbitmq, redis]
-          search-service: [elasticsearch7]
-
-          exclude:
-          # Add combinations to this list that should be excluded from the final
-          # build. Doing this will help keeping the number of jobs down.
-          # E.g. removing 3.8 - min combination will avoid 5 jobs to be submited
-          # [3.8] * [min] * [postgresql10, postgresql13, mysql5, mysql8] * [elasticsearch6]
-          - python-version: 3.8
-            requirements-level: min
-
-          - python-version: 3.9
-            requirements-level: min
-
-          - python-version: "3.10"
-            requirements-level: min
-
-          - db-service: postgresql13
-            python-version: 3.6
-
-          - db-service: mysql8
-            python-version: 3.6
-
-          - search-service: elasticsearch7
-            python-version: 3.6
-
-          - search-service: elasticsearch6
-            db-service: postgresql13
-
-          - search-service: elasticsearch6
-            db-service: mysql8
-
-          - search-service: elasticsearch7
-            db-service: postgresql10
-
-          - search-service: elasticsearch7
-            db-service: mysql5
-
+          python-version: ["3.9", "3.10"]
+          requirements-level: [pypi]
+          db-service: [postgresql13]
+          search-service: [opensearch2,elasticsearch7]
           include:
-          - db-service: postgresql10
-            DB_EXTRAS: "postgresql"
-
-          - db-service: postgresql13
-            DB_EXTRAS: "postgresql"
-
-          - db-service: mysql5
-            DB_EXTRAS: "mysql"
-
-          - db-service: mysql8
-            DB_EXTRAS: "mysql"
+          - search-service: opensearch2
```

Comment:
Only kept search-related extras, since for db we install all database drivers anyways in `invenio-db`